src/python/pants/option/native_options.py

```diff
@@ -138,3 +138,6 @@ def is_enum(typ):
                 val = option_type(val)
 
         return (val, rank)
+
+    def get_unconsumed_flags(self) -> tuple[str, ...]:
+        return tuple(self._native_parser.get_unconsumed_flags())
```

Comment:
Potentially silly question... but, AFAICT, this doesn't seem to be called?

That is, I think this is calling the `src/python/pants/engine/internals/native_engine.pyi` / `src/rust/engine/src/externs/options.rs` PyO3-Rust method, which calls the `src/rust/engine/options/src/lib.rs` fully-Rust method... but there's no calls to this Python one? Am I missing something?

Reply:
There are no calls to it... yet :)

That will come in the switchover change.

Reply:
Ah, right I see, this is just prep work. Thanks.